.github/workflows/build.yml

```diff
@@ -0,0 +1,80 @@
+name: Build

```

Comment:
Reflect this change to the workflow file as well:

```suggestion
name: Build PR

```

Reply:
This workflow can be manually dispatched as well. **Build** fits the name.